cli/internal/run/global_hash_rust.go

```diff
@@ -0,0 +1,26 @@
+//go:build rust
+// +build rust
```

Comment:
are we still running integration tests for both go and rust?

Reply:
I think just Rust, but not sure, cc @gsoltis

Reply:
All of the builds should be using the rust tag now